src/domain/event/EventPage.tsx

```diff
@@ -36,17 +36,20 @@ const EventPage = ({ event, children, success }: EventProps) => {
       >
         {success}
         <div className={styles.backButtonWrapper}>
-          <Button
-            aria-label={t('common.backButton.label')}
-            className={styles.backButton}
-            onClick={() => history.goBack()}
-          >
-            <Icon
-              src={backIcon}
-              className={styles.backButtonIcon}
-              alt={t('common.backButton.label')}
-            />
-          </Button>
+          <div className={styles.backButtonInnerWrapper}>
+            <Button
+              variant="secondary"
+              aria-label={t('common.backButton.label')}
+              className={styles.backButton}
+              onClick={() => history.goBack()}
+            >
+              <Icon
+                src={backIcon}
+                className={styles.backButtonIcon}
+                alt={t('common.backButton.label')}
```

Comment:
Do you need alt here if your button has an aria-label (or vice versa)?

Reply:
No, removed.